composer.json

```diff
@@ -24,6 +24,8 @@
         "php": "^7.3 || ^8.0",
         "cocur/slugify": "^3.0 || ^4.0",
         "doctrine/persistence": "^1.3.6 || ^2.0",
+        "knplabs/knp-menu": "^3.0",
+        "knplabs/knp-menu-bundle": "^3.0",
```

Comment:
if the admin bundle is on require dev, this should be too on dev right?

Reply:
Thought it should be a dependency because it's used in the src https://github.com/sonata-project/SonataClassificationBundle/blob/master/src/Resources/views/CategoryAdmin/list_tab_menu.html.twig#L1

But I moved it to require-dev if preferred